src/porepy/numerics/darcy.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import scipy.sparse as sps
+
+from porepy.numerics.fv import tpfa, source, fvutils
+from porepy.grids.grid_bucket import GridBucket
+from porepy.params import bc, tensor
+from porepy.params.data import Parameters
+from porepy.viz.exporter import export_vtk
+
+
+class Darcy():
```

Comment:
Why does this not inherit from parabolicSolver? Because of no time stepping?

Reply:
yes. 

You could imagine that it inherited from ParabolicProblem, and gave out a zero time-stepping lhs and rhs. But maybe it would be better to add a new class called Elliptic problem which it inherits from (or just call it Elliptic instead of Darcy)?

Reply:
Yes, call it elliptic instead of Darcy. Will inheritance make sense, e.g. avoid cut and paste?